lms/djangoapps/ccx/plugins.py

```diff
@@ -31,6 +33,14 @@ def is_enabled(cls, course, user=None):
             # If ccx is not enable do not show ccx coach tab.
             return False
 
+        if configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False):
+            if not run_extension_point(
+                'PCO_IS_LICENSED_AND_ADMIN',
+                user=user,
+                master_course=course.id,
+            ):
```

Comment:
What about using an `AND` to join these two clauses and add a comment to describe what are we doing here?.  @anfbermudezme 
```suggestion
        if configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and not run_extension_point(..):
        # If course licensing is disable, then regular ccxs are disabled.
```